### PR TITLE
Specify minimum supported Ruby version

### PR DESCRIPTION
**Pull request by jmbehr**
Specify a minimum supported Ruby version (2.3.0) and support it.

### CODE REVIEW

**Review by JacoBriers — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -71,3 +71,23 @@ DogCollar currently provides two formatters.
...
+bundle exec appraisal rails-4 rspec
+bundle exec appraisal rails-5 rspec
+bundle exec appraisal rails-6 rspec # Only Ruby 2.5.0 >= supported
+```
+
+Or to run all appraisals (ensure you're using bundler < 2.0 if using Ruby <
+2.5.0, as these versions will test against Rails 4).
+
+```
+bundle isntall
```
Comment (abrophy) [inline README.md]: typo

Comment (abrophy) [reply README.md]: (not important)

File: lib/dog_collar/contrib/rails/tagged_logger.rb [hunk 2]
```diff
@@ -1,30 +1,53 @@
 # frozen_string_literal: true
 
 require 'dog_collar/logging/delegator'
+require 'forwardable'
```
Comment (abrophy) [inline lib/dog_collar/contrib/rails/tagged_logger.rb:4]: nice

File: lib/dog_collar/contrib/rails/tagged_logger.rb [hunk 3]
```diff
@@ -36,20 +59,10 @@ def tagged(*tags, &block)
...
         end
 
-        protected
-
-        def push_tags(*tags)
-          tags = tags.flatten.reject(&:blank?)
-          logger.current_tags.concat(tags)
-          tags
-        end
+        def_delegators :formatter, :push_tags, :pop_tags, :clear_tags!
```
Comment (abrophy) [inline lib/dog_collar/contrib/rails/tagged_logger.rb:62]: noice

**Review by abrophy — APPROVED** [review]

**Pull request merged**